onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -199,6 +203,12 @@ void InitializeSession(InferenceSession* sess) {
   }
 #endif
 
+#ifdef USE_TENSORRT
```

Comment:
will it cause problems if both CUDA and TensorRT providers are registered?
i'm thinking if one subgraph is partitioned to CUDA and another to TensorRT but both are using the same underlying cuda device.

Reply:
I tested CUDA fallback for TensorRT. It seems no problem.